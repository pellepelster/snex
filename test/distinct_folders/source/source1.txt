source 1.1
<!-- snippet:snippet1 -->
old content
<!-- /snippet:snippet1 -->
source 1.2
<!-- snippet:snippet2 -->
<!-- /snippet:snippet2 -->
source 1.3
