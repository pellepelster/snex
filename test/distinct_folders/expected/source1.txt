source 1.1
<!-- snippet:snippet1 -->
snippet 1 content
<!-- /snippet:snippet1 -->
source 1.2
<!-- snippet:snippet2 -->
snippet 2 content
<!-- /snippet:snippet2 -->
source 1.3
<!-- file:file1.txt -->
file 1 content
<!-- /file:file1.txt -->
source 1.4